spec/presenters/route_summary_card_data_presenter_spec.rb

```diff
@@ -58,8 +60,24 @@
       end
     end
 
+    context "with a route with no answer_value" do
+      let(:routing_conditions) do
+        [
+          build(:condition, id: 1, routing_page_id: 1, check_page_id: 1, answer_value: "Yes", goto_page_id: 2, skip_to_end: false),
+          build(:condition, id: 2, routing_page_id: 2, check_page_id: 1, answer_value: nil, goto_page_id: nil, skip_to_end: true),
+        ]
+      end
```

Comment:
This test doesn't seem quite right; with the two conditions built here, they wouldn't both end up in page 1's `routing_conditions` attribute, would they?

Reply:
Thats a good point!